components/status/StatusPoll.vue

```diff
@@ -39,6 +40,21 @@ async function vote(e: Event) {
   await client.value.v1.polls.$select(poll.id).votes.create({ choices })
 }
 
+async function refresh() {
+  loading.value = true
```

Comment:
Thanks for the updates!

Before merging, I noticed that "Update poll" button can be clicked multiple times in a short period.

The below change will be able to prevent a new request until Elk receives the response from the server.

```suggestion
  if (loading.value) {
    return
  }

  loading.value = true
```